### ISSUE TITLE
Line numbers?

### ISSUE THREAD

**Issue by Efreak** [pgengler/pinot#28]
A column of line numbers at left would be nice. The existing line counter isn't quite as useful. Being able to mark an arbitrary location then go back to it would also be useful. You can do this with existing line numbers, but that doesn't help if before you go back to it you insert more lines.

**Comment by faissaloo** [comment]
Just fyi, line numbers are available in mainline GNU Nano now.

**Comment by pgengler** [comment]
Yeah, this is definitely something I want to get in ... I haven't been keeping `pinot` updated with changes from upstream `nano` for a while but it's something I'll hopefully get back to doing.
